### PR TITLE
ConfigGen: Scala code replaced with Java equivalent

### PR DESCRIPTION
**Pull request by ollivir**
This PR includes a reference copy of `allfeatures.def` converted by the previous version of the generator and a unit test that verifies that the output is exactly identical.

### CODE REVIEW

**Review by bjorncs — APPROVED** [review]

**Comment by bjorncs** [comment]
FYI @gjoranv

File: configgen/src/main/java/com/yahoo/config/codegen/BuilderGenerator.java [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+import static java.util.Arrays.stream;
+
+/**
+ * @author gjoranv
+ * @author ollivir
+ */
+
+public class BuilderGenerator {
+    public static String getBuilder(InnerCNode node) {
+        return getDeclaration(node) + "\n" + //
```
Comment (gjoranv) [inline configgen/src/main/java/com/yahoo/config/codegen/BuilderGenerator.java]: What's the deal with the trailing `//`?

**Review by gjoranv — COMMENTED** [review]
You could have saved some work and made it cleaner with a little helper like this:
```
    public static String joinLines(CharSequence... lines) {
        return String.join("\n", lines);
    }
```

File: configgen/src/main/java/com/yahoo/config/codegen/BuilderGenerator.java (hunk 1 above)
Comment (ollivir) [reply configgen/src/main/java/com/yahoo/config/codegen/BuilderGenerator.java]: Maybe a bit of Eclipse heritage showing. The default formatter in Eclipse will rejoin lines if they fit, so when you have a long concatenation like a template the // is to maintain the line splits.

**Pull request merged**